kolibri/plugins/pdf_viewer/assets/src/views/RecyclableScroller.vue

```diff
@@ -0,0 +1,839 @@
+<template>
```

Comment:
Could you add a comment with the link to the Github URL for the original file (pinned to the specific commit it was copied from if possible)? That will help if we need to update it in the future!

Also, can you flag with code comments where it has been modified from the original and the reason for that modification? [Here is an example](https://github.com/ThEditor/kolibri/blob/3b6477672cb4b4339f0c7a600aa44d2eec96c3ae/kolibri/plugins/pdf_viewer/assets/src/utils/text_layer_builder.js#L21).

Reply:
I was planning to add a comment with a link to the original file, but I had second-thoughts on the format lol. I'll add that now.

The modification is only one line for now, but there might be a lot of changes if I try to implement one of the ideas I have for lazy-loading page sizes. So, I'll add the "where" of modifications later.